src/orders-v2/status-pill.tsx

```diff
@@ -0,0 +1,39 @@
+import { PillStatusType } from "@dashboard/misc";
+import { Box, Text } from "@saleor/macaw-ui-next";
+import React from "react";
+
+// TODO: move to MacawUI
+export const StatusPill = ({
+  status,
+  children,
+}: {
+  status: PillStatusType;
+  children: React.ReactNode;
+}) => {
+  const getFillColorsBasedOnStatus = (status: PillStatusType) => {
+    switch (status) {
+      case "success":
+        return {
+          __backgroundColor: "oklch(94% 0.0898 164deg)",
+          __borderColor: "oklch(79% 0.204 153deg)",
+        };
+    }
+  };
```

Comment:
small optimization but if you move it outisde react component, it will not get re-instantionated on every react render